CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## VERSION 2.3.0 (Unpublished)
+_10_02_2022_
+- Fix - Adjust the tracks
```

Comment:
let's try to modify the changelog in another branch :P

Reply:
done